bitnami/harbor/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: harbor
-version: 2.3.1
+version: 2.4.0
```

Comment:
This PR just fixes an issue but it doesn't add any new feature. Therefore, I think we can simply bump the fix version.